README.md

```diff
@@ -1,25 +1,60 @@
 # ipv-cri-toy-front
 
-[![Github Action: Scan repository](https://github.com/govuk-one-login/ipv-cri-toy-front/actions/workflows/scan-repo.yml/badge.svg?branch=main)](https://github.com/govuk-one-login/ipv-cri-toy-front/actions/workflows/scan-repo.yml?query=branch%3Amain)
-[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=ipv-cri-toy-front&metric=coverage)](https://sonarcloud.io/summary/overall?id=ipv-cri-toy-front)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=ipv-cri-toy-front&metric=coverage)](https://sonarcloud.io/summary/new_code?id=ipv-cri-toy-front)
```

Comment:
```suggestion
[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=ipv-cri-toy-front&metric=coverage)](https://sonarcloud.io/summary/overall?id=ipv-cri-toy-front)
```
Link to the coverage data for the whole repo - new code only includes changes since the last commit.